ci/scripts/deterministic-e2e-test.sh

```diff
@@ -5,30 +5,35 @@ set -euo pipefail
 
 source ci/scripts/common.env.sh
 
+apt-get -y install parallel
```

Comment:
How about make it pre-installed in the CI image?

Reply:
+1, just move it in Dockerfile

Reply:
Should I do something else like building and publishing a docker image?